src/test/java/com/egov/core-services/pretests/collectionServicePretest.feature

```diff
@@ -0,0 +1 @@
+Feature: Update user profile
```

Comment:
collectionService belong to Business-Services. As discussed, create new folder like core-services